apps/src/templates/button.module.scss

```diff
@@ -0,0 +1,254 @@
+@import "color.scss";
+
+// Note: Keep these constants in sync with Button.jsx.
+$button-height-default: 34px;
+$button-height-large: 40px;
+$button-height-narrow: 40px;
+$button-height-small: 20px;
+
+div,
+button,
+a,
+a:link,
+a:visited {
+  &.main {
```

Comment:
i had to make various changes to the css to get it to apply correctly on top of the site-wide styles, since it's no longer using inline styling. see comments and commit history on this PR for more info.